src/ptvsd/adapter/messages.py

```diff
@@ -153,6 +154,7 @@ def initialize_request(self, request):
     def _debug_config(self, request):
         assert request.command in ("launch", "attach")
         self._shared.start_method = request.command
+        self._shared.arguments = request.arguments
```

Comment:
I don't think these need to be shared - `_configure` is invoked by `launch_request` and `attach_request`, and they can just pass the request object to it.